dll/win32/shell32/wine/shpolicy.c

```diff
@@ -435,7 +437,7 @@ static POLICYDATA sh32_policy_table[] =
     strForceCopyACLW,
     SHELL_NO_POLICY
   },
-#if (NTDDI_VERSION < NTDDI_LONGHORN)
+#if (NTDDI_VERSION < NTDDI_VISTA)
```

Comment:
Technically, while it's true the values of NTDDI_LONGHORN and NTDDI_VISTA are the same, the intent of using e.g. NTDDI_LONGHORN originally meant to say that the said feature was present in the so-called Longhorn version of Windows, i.e. before it was renamed into Windows Vista (while using Vista would mean preferrably a feature that was added "late" into e.g. some Vista beta).